specification/contosowidgetmanager/resource-manager/Contoso.Widget/readme.md

```diff
@@ -33,7 +33,7 @@ These settings apply only when `--tag=package-2021-11-01` is specified on the co
 
 ```yaml $(tag) == 'package-2021-11-01'
 input-file:
-  - Microsoft.Contoso/stable/2021-11-01/contoso.json
+  - WidgetManager/stable/2021-11-01/contoso.json
```

Comment:
```suggestion
  - widget/stable/2021-11-01/contoso.json
```
this should be enough? because the resource manager already has the meaning of mgmt plane.

Reply:
@JeffreyRichter recommended exactly this folder structure:

> The are 2 Contoso services: WidgetManagement (control-plane) and Widget (data-plane)
> The RP Namespace is Contoso.Widget
>
> Under data-plane, the folder should be "Widget"
> Under resource-manager, the immediate folder is the RP Namespace: "Contoso.Widget"
> Under this "Contoso.Widget" folder is the control-plane service name folder: "WidgetManagement"

https://github.com/Azure/azure-rest-api-specs/pull/34617#pullrequestreview-2838069034

Reply:
Here is my thinking for this Example folder structure:
- Contoso is the company brand name; the quivalent of Azure.
- And Widget is the "overall service" which has a 
  - Control-plane service: Contoso Widget Management (we don't repeat Contoso) hence WidgetManagement
  - Data-plane service: Contoso Widget (we don't repeat Contoso) hence Widget

- The RP namespace is Contoso.Widget for the company name (instead of Microsoft.*) and Widget being the "overall service" name